### PR TITLE
better way to get an absolute path

### PR DESCRIPTION
**Pull request by RalfJung**
suggested by @matklad

### CODE REVIEW

File: cargo-miri/bin.rs [hunk 1]
```diff
@@ -259,19 +259,8 @@ fn setup(subcommand: MiriCommand) {
     let rust_src = match std::env::var_os("XARGO_RUST_SRC") {
         Some(val) => {
             let path = PathBuf::from(val);
```
Comment (matklad) [inline cargo-miri/bin.rs:261]:
```suggestion
        Some(path) => {
```

Comment (RalfJung) [reply cargo-miri/bin.rs:261]: Oh, it auto-converts?

**Pull request merged**